Dockerfile

```diff
@@ -53,22 +52,21 @@ RUN chmod +x /usr/local/bin/wrapdocker
 
 # Define additional metadata for our image.
 VOLUME /var/lib/docker
-CMD ["wrapdocker"]
```

Comment:
`docker ps` seems to work inside the docker container even without running this wrapper script so I feel more comfortable leaving it out. It seems to be a wrapper to start docker, which won't work in the `Dockerfile` anyways since you can only run one command at the end of the `Dockerfile` to be run in the container. 

As an alternative, I can set `wrapdocker` as the process to start in `supervisor` instead of Docker itself, which will ensure that `wrapdocker` runs (and thus will start Docker). Is this preferred?